napari/layers/points/_tests/test_points.py

```diff
@@ -1988,24 +1906,6 @@ def test_to_mask_2d_with_size_4_bottom_right():
     np.testing.assert_array_equal(mask, expected_mask)
 
 
-def test_to_mask_2d_with_diff_sizes():
-    points = Points([[2, 2], [1, 4]], size=[[1, 1], [2, 2]])
```

Comment:
I think this test should pass if you make `size=[1, 2]`.

Reply:
Oh, right, I thought this was checking anisotropic sizes!